README.md

```diff
@@ -255,36 +255,37 @@ In diesen Ordner wird die __Datenbank__ des Containers _persistiert_, also dauer
             php.ini
             ssmtp.conf
 
-Im `docker/`-Ordner befindet sich die __Konfiguration für die Container__, die wir benutzen, nämlich `mailhog`, `mysql/` und `php-apache/`. Diese enthalten jeweils ein `Dockerfile`, die die Bauanleitungen enthalten, mit der jeweils aus einem _Image_ ein lauffähiger _Container_ gebaut wird.
+In the `docker/` folder is the __configuration for the containers__ we use, namely `mailhog`, `mysql/ ` and `php-apache/ `. These each contain a `dockerfile`, which contains the construction manuals, with which an executable _Container_ is built from an _Image_.
 
-Die Dockerfiles für Mailhog und MySQL sind ganz schlicht, denn sie enthalten lediglich die Angabe, welches Image verwendet wird, ohne dass dieses dann weiter angepasst wird. Das PHP-Apache-Dockerfile ist aufwendiger: Hier bestimmen wir erst das Image, schicken aber einige Anpassungen hinterher. Zum Beispiel aktivieren wir Apache-Module und installieren PHP-Extensions, die REDAXO benötigt. Im Anschluss prüfen wir, ob unser Webroot — dazu gleich mehr! — noch leer ist, und falls es das ist, holen wir uns ein frisches REDAXO von GitHub und entpacken es in den Webroot.
+The Dockerfiles for Mailhog and MySQL are quite simple, because they contain only the indication, which image is used, without this then further adapted. The PHP Apache Dockerfile is more elaborate: Here we first determine the image, but send some adjustments afterwards. For example, we enable Apache modules and install PHP extensions that REDAXO needs. Afterwards we check, if our Webroot - more about that! - is still empty, and if it is, we get a fresh REDAXO from GitHub and unpack it into the webroot.
 
-Die anderen Dateien enthalten Setup-Skripte, Konfigurationen für PHP, Apache und die Datenbank.
+The other files contain setup scripts, configurations for PHP, Apache, and the database.
```

Comment:
The other folders contain setup scripts, configurations for PHP, Apache, and the database.